packages/core/admin/admin/src/pages/Settings/pages/ApplicationInfo/ApplicationInfoPage.tsx

```diff
@@ -95,6 +111,10 @@ const ApplicationInfoPage = () => {
     return null;
   }
 
+  if (!AIUsageData) {
+    return null;
```

Comment:
Not sure but could this mean the whole compo returns null in CE? As <AIUageDataCE> is null ?